src/test/java/uk/gov/justice/hmpps/offenderevents/subscribe/PrisonerEventsListenerTest.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.justice.hmpps.offenderevents.subscribe;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.json.JsonTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import uk.gov.justice.hmpps.offenderevents.model.OffenderEvent;
+import uk.gov.justice.hmpps.offenderevents.services.HMPPSDomainEventsEmitter;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.Mockito.verify;
+
+@JsonTest
+class PrisonerEventsListenerTest {
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @MockBean
+    private HMPPSDomainEventsEmitter eventsEmitter;
+
+    @Captor
+    private ArgumentCaptor<OffenderEvent> offenderEventArgumentCaptor;
+
+    private PrisonerEventsListener listener;
+
+    @BeforeEach
+    void setUp() {
+        listener = new PrisonerEventsListener(objectMapper, eventsEmitter);
+    }
+
+    @SneakyThrows
+    @Test
+    @DisplayName("Will pass offender event to events emitter")
+    void willPassOffenderEventToEventsEmitter() {
+        listener.onPrisonerEvent("""
+            {
+              "Type" : "Notification",
+              "MessageId" : "670250d8-9806-5670-bdbb-09e1ede37855",
+              "TopicArn" : "arn:aws:sns:eu-west-2:754256621582:cloud-platform-Digital-Prison-Services-f221e27fcfcf78f6ab4f4c3cc165eee7",
+              "Message" : "{\\"eventType\\":\\"OFFENDER_MOVEMENT-RECEPTION\\",\\"eventDatetime\\":\\"2021-06-08T14:41:11.526762\\",\\"offenderIdDisplay\\":\\"A5194DY\\",\\"bookingId\\":1201234,\\"movementSeq\\":11,\\"nomisEventType\\":\\"OFF_RECEP_OASYS\\"}",
```

Comment:
Can we use that fancy new triple quote thing?

Reply:
It is already but this is nested JSON inside JSON